django/contrib/admin/views/autocomplete.py

```diff
@@ -40,7 +60,12 @@ def get_paginator(self, *args, **kwargs):
 
     def get_queryset(self):
         """Return queryset based on ModelAdmin.get_search_results()."""
+        limit_choices_to = self.field.remote_field.limit_choices_to
```

Comment:
Isn't this and the following lines essentially the same as `limiti_choices_to = self.field.get_limit_choices_to()`?